enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -714,7 +714,8 @@ You can provide a theme object to the `MetabaseProvider` to customize the look a
 Here is the full list of theme properties supported. All of them are optional.
 
 ```ts
-const theme = {
+import {defineEmbeddingSdkTheme} from "@metabase/embedding-sdk-react";
+const theme = defineEmbeddingSdkTheme({
     // Specify a font to use from the set of fonts supported by Metabase.
     // You can set the font to "Custom" to use the custom font
```

Comment:
Is this going to be weird for clients? Maybe we should even change the name from "custom" to something else (maybe "metabase instance font" and transform it on the FE when we're parsing the config

Reply:
I found it confusing too, but I think that's what they select in the UI on metabase too though.
It should pickup that by default anyway though if it's configured on their instance